### DISCUSSION TITLE
What is the difference between Model

### DISCUSSION THREAD

**Discussion by A380Coding** [jomjol/AI-on-the-edge-device#1746] [Q&A] [answered]
I wonder what is the difference between dig-class100-0150_s2_q and dig-class100-0140_s2_q and is there a meaning to the name?
I read [https://github.com/jomjol/AI-on-the-edge-device/wiki/Choosing-the-Model](url) but could not find the answer

Is 0150 and 0140 a version number?
Just wondering since 0150 detects "5" where it actually is "3" and the number is clear and sharp, no reflections etc.
With 0140 it seems to work better, but I have no idea what the difference between these models is.

**Comment by ghost** [comment, marked as answer]
Look at https://github.com/jomjol/AI-on-the-edge-device/wiki/Neural-Network-Types#2-naming-convention. It's the version number. 1.5.0 or 1.4.0

We get sometimes new images with 
<img width="79" alt="image" src="https://user-images.githubusercontent.com/412645/210277274-c8992b74-bd19-4cd7-a1a1-84d15d9e7eca.png">
for example. 

Because the dataset is mostly from jomjol and me, it has not much different full datasets of other meters. And a set of new images can change much as long as we have not enough images.

So if 1.4 runs fine for you. Use it. Or collect images https://github.com/jomjol/AI-on-the-edge-device/wiki/Learn-models-with-your-own-images and share it.

**Reply by ghost** [reply]
Or you try the https://github.com/haverland/Tenth-of-step-of-a-meter-digit/releases/tag/v1.6.0 dig-class-0160_s2_q.tflite

**Reply by A380Coding** [reply]
Thanks for explanation and link. I missed the other wiki page. :(
I just downloaded 1.6.0 and will give it a try. With 1.5 and 1.4 I'm quite happy so far. I managed to install the ESPs so that there are no reflections on the numbers.
